ddtrace/appsec/_iast/taint_sinks/weak_hash.py

```diff
@@ -1,19 +1,16 @@
 import os
 import sys
-from typing import TYPE_CHECKING  # noqa:F401
 from typing import Any
 from typing import Callable
 from typing import Set
-from typing import Text  # noqa:F401
+from typing import Text
```

Comment:
Text is deprecated. Since python 3, it's an alias to str
https://docs.python.org/3/library/typing.html#typing.Text